lib/Rex/User/Linux.pm

```diff
@@ -359,6 +362,57 @@ sub get_user {
   );
 }
 
+sub lock_password {
+  my ( $self, $user ) = @_;
+
+  # Is the password already locked?
+  my $result = i_run "passwd --status $user";
+
+  die "Unexpected result from passwd: $result"
+    unless $result =~ /^$user\h+(L|NP|P)\h+/;
```

Comment:
Yep, I chose `\s` for now, as that should also match both space and tab - it looks good so far :+1: